### PR TITLE
Port tetris from NetBSD

### PR DESCRIPTION
**Pull request by j-piecuch**
`tetris/input.c` needed to be modified so that it doesn't use `poll()`.

### CODE REVIEW

File: bin/tetris/input.c [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  signal(SIGALRM, handle_sigalrm);
+}
+
+static int read_one(char *c) {
+  if (have_char) {
+    have_char = 0;
+    *c = buffered_char;
+    return 1;
+  }
+  return read(STDIN_FILENO, c, 1);
```
[the remark this answers is not in the document]
Comment (j-piecuch) [reply bin/tetris/input.c]: You're right, we should always read the buffered char after calling `rwait()`. I have fixed it.

**Review by franciscozdo — APPROVED** [review]

**Review by wiklam — APPROVED** [review]
LGTM, review focused on file bin/tetris/input.c.

File: sys/tests/callout.c [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #include <sys/klog.h>
 #include <sys/libkern.h>
 #include <sys/callout.h>
+#include <sys/time.h>
```
Comment (cahirwpz) [inline sys/tests/callout.c:4]: A leftover after some change?

Comment (j-piecuch) [reply sys/tests/callout.c:4]: Yes.

**Review by cahirwpz — APPROVED** [review]

**Pull request merged**